style.css

```
@@ -0,0 +1,111 @@
+main {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  /* grid-template-columns: 1fr 1fr 1fr; */
+}
+
+article {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.container {
+  display: grid;
+  /* grid-template-columns: repeat(9, 1fr);
+  grid-template-rows: repeat(9, 1fr); */
+  width: 100%;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+  grid-gap: 4px;
+  background-color: black;
+  margin: 0;
+  padding: 4px;
+  box-sizing: border-box;
+}
+
+.cell {
+  list-style: none;
+  background-color: white;
+  /* border: 1px solid black; */
+  display: flex;
+  height: 3rem;
+  width: 3rem;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.clickable {
+  cursor: pointer;
+}
+
+.clickable:hover {
+  -moz-box-shadow: inset 0 0 10px #686767;
+  -webkit-box-shadow: inset 0 0 10px #686767;
+  box-shadow: inset 0 0 10px #686767;
+}
+
+.flaggable:hover {
+  -moz-box-shadow: inset 0 0 10px #a70000;
+  -webkit-box-shadow: inset 0 0 10px #a70000;
+  box-shadow: inset 0 0 10px #a70000;
+}
+
+.hide {
+  display: none;
+}
+.revealed {
+  -moz-box-shadow: inset 0 0 10px #000000;
+  -webkit-box-shadow: inset 0 0 10px #000000;
+  box-shadow: inset 0 0 10px #000000;
+  width: 100%;
+  height: 100%;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 1.3em;
+  margin: 0;
+  padding: 0;
+}
+
+.flag {
+  background-color: red;
+  color: brown;
+}
+
+.game-options {
+  display: flex;
+  justify-content: flex-end;
+  background-color: #000000;
```

fact check me: prefer 3 digit hex codes for simple stuff like this
```suggestion
  background-color: #000;
```